### DISCUSSION TITLE
Asking help to view uploaded files

### DISCUSSION THREAD

**Discussion by Mareeswari-1326** [community/community#54692] [Repositories]
### Select Topic Area

General

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

 I'm Mareeswari J. I've been using Github for 2 months. I uploaded some of my pdf files into my Github repository. But I'm facing a problem while opening the pdf file. It Shows like 'Unable to render rich display'. If I try to view it after downloading the file, it shows like 'This site can't be reached'. So please help me to rectify the error messages I got. I hope you will help me.
Thanking You

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by airtower-luna** [comment]
"This site can't be reached" sounds like a network problem. For which domain do you get that?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Mareeswari-1326 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
